STATISTICS.md

```diff
@@ -183,7 +183,8 @@ eof_offset | int gauge | | Last PARTITION_EOF signaled offset
 lo_offset | int gauge | | Partition's low watermark offset on broker
 hi_offset | int gauge | | Partition's high watermark offset on broker
 ls_offset | int gauge | | Partition's last stable offset on broker, or same as hi_offset is broker version is less than 0.11.0.0.
-consumer_lag | int gauge | | Difference between (hi_offset or ls_offset) - max(app_offset, committed_offset). hi_offset is used when isolation.level=read_uncommitted, otherwise ls_offset.
+consumer_lag | int gauge | | Difference between (hi_offset or ls_offset) - committed_offset). hi_offset is used when isolation.level=read_uncommitted, otherwise ls_offset.
```

Comment:
extra parenthesis after committed_offset.

also, `-` means difference, so better to replace that with 'and'. same comment for consumer_lag_stored.